contrib/packaging/rpm/cfg/Dockerfile

```diff
@@ -1,24 +1,13 @@
-FROM fedora:24
+FROM fedora:25
 
-MAINTAINER "Andre Martins <andre@cilium.io>"
+LABEL "Maintainer: Marcin Skarbek <marcin@skarbek.name>"
 
-RUN dnf install -y gettext curl gcc fedora-packager fedora-review && \
-cd /tmp && \
-curl -Sslk -o go1.7.4.linux-amd64.tar.gz \
```

Comment:
Why opt for fedora golang package instead direct download? We currently don't support 1.8 and fedora can update it to 1.8 later on, no?